torch/onnx/symbolic_opset13.py

```diff
@@ -888,9 +888,9 @@ def quantized_linear(
     return symbolic_helper.quantize_helper(g, output, op_scale, op_zero_point)
 
 
-@_onnx_symbolic("quantized::conv2d")
+@_onnx_symbolic("quantized::conv1d_relu")
 @_beartype.beartype
-def quantized_conv2d(
+def quantized_conv1d_relu(
```

Comment:
This seemingly odd 1d->2d diff is just from copying the order used in `symbolic_opset10.py`. This `quantized_conv2d` is unchanged and shows up below.